eawx-build/Configuration/v1/BuildComponentFactory.cs

```diff
@@ -23,9 +25,10 @@ public BuildComponentFactory(ILogger logger = null)
 
         public ITaskBuilder Task(string taskTypeName) {
             return taskTypeName switch {
-                "RunProgram" => new RunProcessTaskBuilder(),
-                "Clean" => new CleanTaskBuilder(),
-                "Copy" => new CopyTaskBuilder(),
+                "RunProgram" => new RunProcessTaskBuilder(_fileSystem),
```

Comment:
At some point we should probably think about getting awy from the string comparisons here and moving to some sort of enum mapping.